wafer/schedule/templates/wafer.schedule/penta_schedule.xml

```diff
@@ -70,7 +78,8 @@
                         <description/>
                         <type/>
                     {% endif %}
-                    <conf_url>{{ items.item.get_url }}</conf_url>
+                    {# forcing https here is a bit horrible - make this configurable somewhere? #}
+                    <conf_url>https://{{ WAFER_CONFERENCE_DOMAIN }}{{ items.item.get_url }}</conf_url>
```

Comment:
> I'm pretty sure that won't do the right thing for the static site generation script, though.

Ah. That might be something to fix.

I'm OK with using WAFER_CONFERENCE_DOMAIN like this, we do in the e-mail templates, too.